src/store-api/src/path_utils.rs

```diff
@@ -45,10 +45,40 @@ pub fn region_dir(path: &str, region_id: RegionId) -> String {
     )
 }
 
+/// get_storage_path returns the storage path from the region_dir.
+/// The storage path is constructed from the catalog and schema, which are generated by `common_meta::ddl::utils::region_storage_path`.
+/// We can extract the catalog and schema from the region_dir by following example:
+/// ```
+/// use common_meta::ddl::utils::get_catalog_and_schema;
+///
+/// fn catalog_and_schema(region_dir: &str, region_id: RegionId) -> Option<(String, String)> {
+///     get_catalog_and_schema(&get_storage_path(region_dir, region_id)?)
+/// }
+/// ```
+pub fn get_storage_path(region_dir: &str, region_id: RegionId) -> Option<String> {
+    // For example, if region_dir is "data/my_catalog/my_schema/42/42_0000000001/", the parts will be '42/42_0000000001/'.
+    let parts = format!(
+        "{}/{}/",
+        region_id.table_id(),
+        region_name(region_id.table_id(), region_id.region_number())
```

Comment:
Maybe we should remove the suffix so it still works when the dir doesn't end with `/`.
```suggestion
        "{}/{}",
        region_id.table_id(),
        region_name(region_id.table_id(), region_id.region_number())
```